collections/nemo_nlp/nemo_nlp/data/data_layers.py

```diff
@@ -157,6 +144,20 @@ def create_ports():
         }
         return {}, output_ports
 
+    """
```

Comment:
How come class docstrings have been moved from after the class to before the __init__? I think most of our docstrings are currently either after the class or after __init__?